### DISCUSSION TITLE
Account for multiple rulesets when searching for beatmaps

### DISCUSSION THREAD

**Discussion by ticotaco72** [ppy/osu#16469] [Ideas]
Beatmaps from rulesets other than used at the time should be showed in results, or for example number of found beatmaps for given search terms should appear next to the icon of every ruleset.

Thanks.

**Comment by bdach** [comment]
results where? do you mean song select, or the beatmap listing?

also what is the goal of this proposal? what would showing the number achieve?

**Reply by ticotaco72** [reply]
Song select (beatmap listing already shows them all in one place).
As for the goal, you sometimes search for a song which only has a version for 1 ruleset, and you may not even know that you may be looking in the wrong ruleset. If you know that such a beatmap was found somewhere else it would save your time.

**Reply by ticotaco72** [reply]
Of course, the way to show that to user doesn't need to be specifically showing numbers in the described way

**Reply by peppy** [reply]
Can agree with this, in the case there are zero matches I'd like there to be a message showing "Results were found for a different ruleset, click here to view" or similar.
